phabricatoremails/mail.py

```diff
@@ -55,8 +59,39 @@ def encode_from(self, from_address):
             return from_address
 
     def to_mime_message(self, from_address, include_target_in_subject=False):
-        msg = MIMEMultipart("alternative")
+        # There's four email threading headers that we use:
+        # * Thread-Topic and Thread-Index are for Outlook.
+        # * In-Reply-To and References are RFC 2822 compliant and and used by
+        #   almost everything else.
+        # Note: just like Phabricator, we don't track the parent-child relationship of
+        # each message as a long interconnected chain. Instead, we configure each
+        # message such that it "References"/is "In-Reply-To" an imaginary first message
```

Comment:
is the `/is` a typo here or something specific that I am not understanding?

Reply:
You're right that it's unclear - I wanted to group "References"/"In-Reply-To", but I figured that the "is" would help with grammar:
* Each message `References` ...
* Each message **is** `In-Reference-To` ...

This might be easier to understand without the "is" though, yeah.